_posts/2020-10-02-we-support-the-sustainable-development-goals.md

```diff
@@ -0,0 +1,60 @@
+
+Signalen's success can contribute to SDG 11.3: "By 2030, enhance inclusive and sustainable urbanization and capacity for participatory, integrated and sustainable human settlement planning and management in all countries."
+
+### Goal 16: Peace, Justice, and Strong Institutions
+
+The Foundation for Public Code contributes to achieving goal 16 by increasing transparency, quality public service delivery and participation.
+
+An important but often forgotten part of access to information is *how* policy is implemented. Developing public code projects in the open (publishing all code and accompanying documentation), allows the public to see how policy choices that are implemented through digital service provision work in practice. This helps the public understand how this affects their lives, for example algorithms shaped by public policy.
+
+A small practical example of this is work we've done with the [OpenZaak codebase](https://publiccode.net/codebases/openzaak.html). We are helping them move their conversations to open platforms - to GitHub when needed, or to a mailing list where anyone can join or read the archive. This makes it possible for anyone to see what is happening (SDG 16.6) and also participate if they have something to contribute (SDG 16.7).
+
+### Goal 17: Partnerships for the Goals
+
+The Foundation for Public Code supports goal 17 by building networks and making tools available.
+
+Partnerships are the core of what we do. Our concept of [codebase stewardship](https://publiccode.net/codebase-stewardship/) is centered on partnerships. Collaboration and participation by the many is key for sound digital public infrastructure. The Foundation for Public Code is a ‘commons’ organization. One of our greatest values lies in bringing entities together that do not traditionally collaborate, and supporting these with their concrete public code needs.
+
+## In the future
+
+As an ever-increasing number of domains targeted by the Sustainable Development Goals undergo digital transformation, codebase stewardship work such as our own becomes increasingly linked to sustainability overall. Making sure that all SDG initiatives that develop digital tooling take into account the long-term support, iterative improvement, and maintenance of open code should be core to the overall mission of both the Foundation for Public Code and the SDG ecosystem.
+
+We commit to highlighting how we're supporting the SDG in all our major projects in the future, because none of our work is independent or inseparable. We take the Sustainable Development Goals seriously, and believe that awareness of them is in itself important.
```

Comment:
```suggestion
We commit to highlighting how we're supporting the SDG in all our major projects in the future, because none of our work is independent or inseparable. We take the Sustainable Development Goals seriously, and believe that awareness of them is in itself important.
```
```suggestion
We commit to highlighting how we're supporting the SDGs in all our major projects in the future, because none of our work is independent or inseparable. We take the Sustainable Development Goals seriously, and believe that awareness of them is in itself important.
```